packages/rule-tester/src/RuleTester.ts

```diff
@@ -210,7 +210,7 @@ export class RuleTester extends TestFramework {
     // file name (`foo.ts`), don't change the base path.
     if (
       filename != null &&
-      (filename.startsWith('/') || filename.startsWith('..'))
+      (path.isAbsolute(filename) || filename.startsWith('..'))
```

Comment:
This is where CI fails. Even if filename is given as “C://...” we need to change the base path so that the linter can find the config.

Reply:
BTW, the value of messages when Linter does not find a config for a given “filename” is as follows  
```json
[
 { "ruleId": null, "severity": 1, "message": "No matching configuration found for C:\\Code....", "line": 0, "column": 0 }
]
```

I think it would be nice to print an error message when this happens, or have an assertion for that. What do you think? @ronami

Reply:
FWIW I _think_ I'm in favor 🙂